localization/ndt_scan_matcher/include/ndt_scan_matcher/ndt_scan_matcher_core.hpp

```diff
@@ -25,6 +25,7 @@
 
 #include <diagnostic_msgs/msg/diagnostic_array.hpp>
 #include <geometry_msgs/msg/pose_with_covariance_stamped.hpp>
+#include <geometry_msgs/msg/pose_array.hpp>
```

Comment:
by clang-format
```suggestion
#include <geometry_msgs/msg/pose_array.hpp>
#include <geometry_msgs/msg/pose_with_covariance_stamped.hpp>
```

I think that these files need to be arranged in alphabetical order.